### ISSUE TITLE
Firefox doesn't wait page load. ImplicitlyWait doesn't work for findElement.

### ISSUE THREAD

**Issue by zenonwch** [mozilla/geckodriver#530]
## Firefox Version
52.0.0
## Platform
OSX Sierra 10.12.3
## Driver Version
0.15.0
## Selenium Version
javascript binding @3.3.0 with fixes for geckodriver@0.15.0 

## Result
NoSuchElementError: Unable to locate element: *[id="gb_70"]

## Steps to reproduce -
```
let wd = require('selenium-webdriver');
let test = require('selenium-webdriver/testing');
let caps = wd.Capabilities.firefox();
let driver = new wd.Builder().withCapabilities(caps).build();

test.describe('Test', function () {
    test.before(() => driver.get('https://www.google.com/'));

    test.it('Click button', () => driver.findElement(wd.By.id('gb_70')).click());

    test.after(() => driver.quit());
}
```

**Comment by sethuster** [comment]
I can confirm this issue as well.

**Comment by bsolosy-syncronex** [comment]
Yes, I'm seeing this issue as well.
Let me also add:  There has been considerable debate over the years within the Selenium community as to whether implicit waits or explicit waits are preferable.
I really don't care (too much) about which we decide to use.
I just need some kind of wait to work, and right now for me, neither implicit nor explicit waits seem to be  working.  (Or, if they are, then the syntax has somehow changed yet again.)
Another important point worth making:
When waits (either implicit or explicit) stop working, it tends to break tons of selenium automation (worldwide).  Thus, I have a hard time understanding why this very basic feature seems to break so often.
Please stop breaking (or incessantly changing) waits.
Thanks!

**Comment by davidPGH** [comment]
Seen in C# also ..totally  blocked by this I'm afraid

**Comment by snpanigrahi9** [comment]
When this issue will be solved.
Any idea?

**Comment by andreastt** [comment]
Probably duplicate of https://github.com/mozilla/geckodriver/issues/308.
